yb-voyager/src/tgtdb/suites/yugabytedbSuite.go

```diff
@@ -189,6 +189,20 @@ var YBValueConverterSuite = map[string]ConverterFn{
 			return columnValue, nil
 		}
 	},
+	"io.debezium.data.Json": func(columnValue string, formatIfRequired bool) (string, error) {
+		if formatIfRequired {
+			return fmt.Sprintf("'%s'", columnValue), nil
+		} else {
+			return columnValue, nil
+		}
+	},
+	"io.debezium.data.Enum": func(columnValue string, formatIfRequired bool) (string, error) {
+		if formatIfRequired {
+			return fmt.Sprintf("'%s'", columnValue), nil
+		} else {
+			return columnValue, nil
+		}
+	},
```

Comment:
This function definition is repeating multiple times. How about defining a `func` at the top -- named `quoteForUpdateCommands()`? Then you can simply use the function name instead of repeating the function body.